spec/resources/user_resource_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe UserResource do
+
+  let(:model) { User.new }
+  let(:context) { Hash.new }
+
+  let(:resource) { described_class.new(model, context) }
+  subject { resource }
+
+  describe "to-one relationships" do
+    specify "can use non-ActiveRecord associations" do
+      expect(subject.post._model.title).to start_with "Hello"
+    end
+  end
```

Comment:
Unlike to-one relationships, to-many relationships have to be ActiveRecord associations, so that the policy scope can be applied to them. Don't think it's worth testing what happens when they aren't, because the resulting `NoMethodError`s should be pretty explanatory.